pkg/apis/pipeline/v1alpha1/run_types.go

```diff
@@ -189,6 +201,12 @@ func (r *Run) IsCancelled() bool {
 	return r.Spec.Status == RunSpecStatusCancelled
 }
 
+// GetStatusMessage function returns the status message is one is set on the Run
+// If nothing is set as status message, GetStatusMessage returns an empty string.
+func (tr *Run) GetStatusMessage() RunSpecStatusMessage {
```

Comment:
Current `golangci-lint` versions will start barfing on the receiver name here not matching others in the file - so please change this to `func (r *Run) ...`.

Reply:
Done, thanks for pointing that out.